app/helpers/application_helper.rb

```diff
@@ -25,10 +25,10 @@ def getDayFromString (day)
 			weekDayRequested = daysBefore(currentWeekDay, 6)
 		when t(:week_day_sunday)
 			weekDayRequested = daysBefore(currentWeekDay, 0)
-		when t(:week_day_tomorrow)
+		when t(:week_tomorrow)
 			weekDayRequested = 1
 		else
-			weekDayRequested = Date.today.wday
+			return nil
```

Comment:
Don't think nil return is a good solution for every code, but it's a personal opinion. I would return -1(not hardcoded, but as a static value) or something like that.

Feel free to change it or not.